packages/server/api/src/app/database/migrations/1754489349004-AddTriggerSourceToFlowRun.ts

```diff
@@ -0,0 +1,26 @@
+import { FlowRunTriggerSource } from '@openops/shared';
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddTriggerSourceToFlowRun1754489349005
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      ALTER TABLE flow_run 
+      ADD COLUMN IF NOT EXISTS "triggerSource" varchar;
+
+      UPDATE flow_run 
```

Comment:
this is the only thing im not 100% convinced on, should we fill them retroactively? if not we cannot enforce the NOT NULL